### PR TITLE
Upgrade 30 dependencies to the latest version (Apr. 21)

### PR DESCRIPTION
**Pull request by joos2010kj**
List of yarn-upgraded dependencies:

- case-sensitive-paths-webpack-plugin
- font-awesome-sass-loader
- fs-extra
- gh-pages
- isomorphic-fetch
- moment
- node-sass
- normalize-scss
- papaparse
- react-addons-test-utils
- react-copy-to-clipboard
- react-css-modules
- react-file-download
- react-tabs
- react-tabs-redux
- re-reselect
- style-loader
- tether
- throttle-debounce
- tslib
- webpack-notifier
- @types/classnames
- @types/d3-selection
- @types/fetch-mock
- @types/isomorphic-fetch
- @types/json-stable-stringify
- @types/papaparse
- @types/react-css-modules
- @types/throttle-debounce
- @types/webpack-env

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,61 +38,63 @@
   },
   "dependencies": {
     "ajv": "^5.2.2",
+    "codecov": "^3.8.1",
```
Comment (domoritz) [inline package.json:41]: Please remove

File: package.json [hunk 2]
```diff
@@ -38,61 +38,63 @@
...
     "compassql": "0.20.2",
     "font-awesome": "^4.7.0",
-    "font-awesome-sass-loader": "^1.0.3",
-    "isomorphic-fetch": "^2.2.1",
-    "moment": "^2.18.1",
-    "normalize-scss": "^6.0.0",
-    "papaparse": "^4.3.5",
+    "font-awesome-sass-loader": "^2.0.1",
```
Comment (domoritz) [inline package.json:44]: This should be a dev dependency I think

File: package.json [hunk 3]
```diff
@@ -38,61 +38,63 @@
...
+    "isomorphic-fetch": "^3.0.0",
+    "moment": "^2.29.1",
+    "normalize-scss": "^7.0.1",
```
Comment (domoritz) [inline package.json:47]: This should be a dev dependency I think

File: package.json [hunk 4]
```diff
@@ -38,61 +38,63 @@
...
-    "vega-datasets": "^1.8.0",
+    "vega-datasets": "^2.2.0",
     "vega-expression": "^2.3.1",
     "vega-lite": "2.4.0",
-    "vega-typings": "^0.2.15",
     "vega-tooltip": "^0.4.2",
-    "vega-util": "1.7.0"
+    "vega-typings": "^0.2.15",
+    "vega-util": "1.7.0",
+    "webpack-notifier": "^1.13.0"
```
Comment (domoritz) [inline package.json:83]: Why was this added?

File: package.json [hunk 5]
```diff
@@ -38,61 +38,63 @@
...
     "compassql": "0.20.2",
     "font-awesome": "^4.7.0",
-    "font-awesome-sass-loader": "^1.0.3",
-    "isomorphic-fetch": "^2.2.1",
-    "moment": "^2.18.1",
-    "normalize-scss": "^6.0.0",
-    "papaparse": "^4.3.5",
+    "font-awesome-sass-loader": "^2.0.1",
+    "isomorphic-fetch": "^3.0.0",
+    "moment": "^2.29.1",
```
Comment (domoritz) [inline package.json:46]: Can we get rid of this? Moment was useful a few years ago but there are lighter alternatives today.

**Review by domoritz — CHANGES_REQUESTED** [review]

**Comment by domoritz** [comment]
Let me know when this is ready. For now, I'll make it as draft.

File: package.json (hunk 5 above)
Comment (joos2010kj) [reply package.json:44]: What should I replace this moment with? It is marked as a peer dependency of react-moment.

Comment (domoritz) [reply package.json:44]: Leave it for now then but we may not need react moment anymore. We can just use the browser APIs for time.

**Comment by domoritz** [comment]
Remove the yarn error log file.

**Comment by joos2010kj** [comment]
The font issue has been resolved.

File: src/components/app.scss [hunk 6]
```diff
@@ -1,3 +1,5 @@
+$base-font-family: sans-serif !default;
```
Comment (domoritz) [inline src/components/app.scss:1]: Can you remove the !default and move this after the normalize import?

Comment (joos2010kj) [reply src/components/app.scss:1]: Hi,

I have tried to put it AFTER @import initially, but the font change did not kick in.

So I googled a bit, and tried it similar to what this stackoverflow post said (https://stackoverflow.com/questions/19793482/how-can-i-set-basefontfamily-for-bootstrap) by putting BEFORE it, and it worked.

**Comment by domoritz** [comment]
Thank you!
